bundles/org.openhab.automation.jsscripting/README.md

```diff
@@ -21,23 +36,215 @@ log:set DEBUG org.openhab.core.automation
 
 For more information on the available APIs in scripts see the [JSR223 Scripting]({{base}}/configuration/jsr223.html) documentation.
 
-## Script Examples
+The following examples show how to access common openHAB functionalities.
+
+## Logging
 
-JavaScript scripts provide access to almost all the functionality in an openHAB runtime environment.
 As a simple example, the following script logs "Hello, World!".
 Note that `console.log` will usually not work since the output has no terminal to display the text.
+__Please note:__ Support for `console.log` will likely be added together with a logging API in the [helper library](https://github.com/openhab/openhab-js).
 The openHAB server uses the [SLF4J](https://www.slf4j.org/) library for logging.
 
-```js
-const LoggerFactory = Java.type('org.slf4j.LoggerFactory');
-
-LoggerFactory.getLogger("org.openhab.core.automation.examples").info("Hello world!");
+```javascript
+let logger = Java.type('org.slf4j.LoggerFactory').getLogger('org.openhab.rule.' + this.ruleUID);
+logger.info('Hello world!');
+logger.warn('Successfully logged warning.');
+logger.error('Successfully logged error.');
 ```
 
-Depending on the openHAB logging configuration, you may need to prefix logger names with `org.openhab.core.automation` for them to show up in the log file (or you modify the logging configuration).
-
 The script uses the [LoggerFactory](https://www.slf4j.org/apidocs/org/slf4j/Logger.html) to obtain a named logger and then logs a message like:
 
 ```text
-    ... [INFO ] [.openhab.core.automation.examples:-2   ] - Hello world!
+    ... [INFO ] [org.openhab.rule.<ruleId>   ] - Hello world!
+    ... [WARN ] [org.openhab.rule.<ruleId>   ] - Successfully logged warning.
+    ... [ERROR] [org.openhab.rule.<ruleId>   ] - Successfully logged error.
+```
+
+## Core Actions
+
+The openHAB services, which are pre-included in the integrated JavaScript engine, must explicitely be imported.
+
+__Please note:__ The [helper library](https://github.com/openhab/openhab-js) is on the way and will become the preferred API to work with openHAB.
+
+```javascript
+let openhab = require('@runtime');
+```
+
+### itemRegistry
+
+```javascript
+let state = openhab.itemRegistry.getItem(itemName).getState();
+```
+
+You can use `toString()` to convert an item's state to string or `toBigDecimal()` to convert to number.
+
+### Event Bus Actions
+
+```javascript
+openhab.events.sendCommand(itemName, command);
+openhab.events.postUpdate(itemName, state);
+```
+
+`command` and `state` can be a string `'string'` or a number depending on the item.
+
+### Exec Actions
+
+Execute a command line.
+
+```javascript
+openhab.Exec = Java.type('org.openhab.core.model.script.actions.Exec');
+let Duration = Java.type('java.time.Duration');
+
+// Execute command line.
+openhab.Exec.executeCommandLine('echo', 'Hello Wordl!');
```

Comment:
These are great examples, i will probably merge these in when the help library comes with some slight modifications.